app/controllers/articles_controller.rb

```diff
@@ -1,8 +1,11 @@
 class ArticlesController < ApplicationController
+  before_action :authenticate_user!, except: %i[index]
```

Comment:
It is best practice to use `only` everywhere

Reply:
I think that except with only 1 action looks more beautiful. I want to leave this

Reply:
Okay, as you wish)